src/Core/Settings.h

```diff
@@ -796,7 +796,9 @@ static constexpr UInt64 operator""_GiB(unsigned long long value)
     M(Int64, max_join_range, 300, "Max join range", 0) \
     M(Bool, compact_kv_stream, true, "Control if compact a changelog kv or versioned kv stream during query", 0) \
     M(UInt64, keep_versions, 3, "Control how many versions for each key kept in memory when joining. Used in versioned_kv join", 0) \
-    M(Int64, join_latency_threshold, 0, "Control when to start join left stream with right stream. Zero means automatic system picked threshold", 0) \
+    M(Int64, join_latency_threshold, 0, "Control when to start join left stream with right stream (only works on join alignment). Zero means automatic system picked threshold", 0) \
```

Comment:
`Control when to start join left stream with right stream for alignment join scenarios. Zero means system picked threshold.`